### ISSUE TITLE
Personal google maps, in an iframe. Server switcher icons disappear.

### ISSUE THREAD

**Issue by JohnLukeBentley** [Bronkoknorb/Server-Switcher#7]
If I embed a personal google map, in an iframe, in my web page then the server switcher icons disappear (whether on the dev or live side). This does not occur for general google maps (in iframes).

What I do in my environment to trigger the bug.

1. Setup a sever-switcher project as follows:

        PhilorumVirtualHost

        Development sever: http://  local.philorum.org:8080 /
        Live Server:  https:// www.philorum.org  /

2. Create a test file on my dev machine:

        <!DOCTYPE html>
        <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
        <head>
          <meta charset="utf-8" />
          <title>Title</title>
        </head>
        <body>
          <h1>server-switcher-bug-iframe</h1>

          
          <iframe src="https://www.google.com/maps/d/embed?mid=1Un-5Ddbazf_kcYTagJUbx3XENe0" width="425" height="350"></iframe>

          
        </body>
        </html>

3.  Comment out all iframes. Refresh the page. Observe the Dev icon. This ensures Server Switcher is working as normal.
4.  Uncomment the first iframe. Refresh the page. Observe the Dev icon has disappeared. This is unexpected.

That reveals the bug. However, note ....

5. Comment out the first iframe and uncomment the second iframe. Notice the second iframe works: the Dev icon reappears.

The first, buggy, "Personal google Maps" iframe was generated by:
* Visiting https://www.google.com.au/maps, logged in. 
* Click on left hand hamburger menu.
* "Your places" > Maps > [Click on one of my maps:] "Philorum  Central" > Open in my maps ...
* Click on the topmost 3 dot menu > "Embed on my site" >  Copy iframe link.

The second, well behaved, "General Google Maps" iframe was generated by:
* Visiting https://www.google.com.au/maps
* Moving to some location.
* Click on left hand hamburger menu.
* "Share or embed map" > Embed map [tab] > Copy iframe link.

In short, this is really weird bug. I find it difficult to see a relevant possible difference between the two iframes that would cause Server Switcher to fail in the first case but not the second.

Can you reproduce the bug at your end?

**Comment by Bronkoknorb** [comment]
Hi John,
this bug indeed sounds very weird.
Unfortunately I do not have time to work on the Server Switcher project currently.
So either you or someone else would have to debug it, to trace it down.
Pull requests are of course welcome!

**Comment by JohnLukeBentley** [comment]
Hermann, thanks.

Yes I'm a bit time poor for this, also. But I'll put it on my to do list. It'll just be a very long time before I get to it. In other words I'd suggest we leave the issue open, and if some third person wants to jump in and fix it: they'd be welcome to.

By the way, Server Switcher is a very handy tool. I've been using it for years. So thanks for it!
